pkg/controller/migmigration/task.go

```diff
@@ -19,6 +19,8 @@ const (
 	Created                       = ""
 	Started                       = "Started"
 	Prepare                       = "Prepare"
+	Authorization                 = "Authorization"
+	NotAuthorized                 = "NotAuthorized"
```

Comment:
What about `AuthorizationFailed` to match the other phases?

Reply:
`NotAuthorized` was @jortel's recommendation over `AuthorizationFailed`.